src/Weiss/projectile/Projectile.java

```diff
@@ -0,0 +1,46 @@
+package weiss.projectile;
+
+public class Projectile {
+    private final double angle;
+    private final double velocity;
+    public double time;
+
+
+    public Projectile(double angle, double velocity) {
+        this.angle = angle;
+        this.velocity = velocity;
+        this.time = time;
```

Comment:
This line doesn't do anything.
I'm fairly sure that it shows up as a warning as well.